app/views/provider_mailer/set_up_organisation_permissions.text.erb

```diff
@@ -1,13 +1,29 @@
 Dear <%= @provider_user.full_name || 'colleague' %>
 
+<% if @multiple_relationships -%>
+Candidates can now find courses on GOV.UK that you work on with the partner organisations listed below.
+
+You cannot manage applications to these courses until either you or your partner organisations have set up organisational permissions.
+
+<% @relationships_to_set_up.each do |organisation, partner_organisations| -%>
+
+For <%= organisation %>, you need to set up permissions for courses you work on with:
+
+<% partner_organisations.each do |partner_organisation| -%>
```

Comment:
could we fix the indentation here?

Reply:
The issue I found when doing this was that the markdown lists do not render correctly if indented. https://github.com/DFE-Digital/apply-for-teacher-training/pull/5676/commits/4a477dc867d32eda76d91e67538c881fe0f47ba0 simplifies the conditionals a bit.